package.json

```diff
@@ -32,73 +32,67 @@
-    "ember-cli-moment-shim": "^3.7.1",
+    "ember-cli-htmlbars": "^5.7.1",
+    "ember-cli-moment-shim": "^3.8.0",
     "ember-cli-sass": "10.0.1",
-    "ember-compatibility-helpers": "^1.2.5",
     "ember-concurrency": "2.2.0",
     "ember-concurrency-decorators": "2.0.3",
-    "ember-cp-validations": "4.0.0-beta.12",
-    "ember-get-config": "0.2.4",
-    "ember-is-mobile": "2.0.5",
-    "ember-keyboard": "6.0.3",
-    "ember-moment": "8.0.0",
-    "ember-responsive": "3.0.5",
-    "ember-tether": "^1.0.0",
-    "ember-truth-helpers": "2.1.0",
+    "ember-get-config": "0.5.0",
+    "ember-is-mobile": "4.1.2",
+    "ember-keyboard": "6.0.4",
+    "ember-moment": "9.0.1",
+    "ember-resize-observer-service": "^0.4.5",
+    "ember-responsive": "4.0.2",
+    "ember-tether": "^2.0.1",
+    "ember-truth-helpers": "3.0.0",
     "ember-validators": "4.0.0",
-    "fomantic-ui-css": "2.8.3",
+    "fomantic-ui-css": "2.8.8",
     "markdown-it": "8.4.2",
-    "polyfill-array-includes": "^2.0.0",
-    "sass": "1.25.0",
-    "walk-sync": "^2.0.2"
+    "sass": "1.43.5",
+    "walk-sync": "^3.0.0"
   },
   "devDependencies": {
-    "@ember/jquery": "^1.1.0",
     "@ember/optional-features": "^2.0.0",
     "@ember/test-helpers": "^2.4.2",
     "@embroider/test-setup": "^0.43.5",
     "@glimmer/component": "^1.0.4",
     "@glimmer/tracking": "^1.0.4",
     "babel-eslint": "^10.1.0",
-    "broccoli-asset-rev": "3.0.0",
+    "broccoli-asset-rev": "^3.0.0",
     "ember-auto-import": "^1.11.3",
     "ember-cli": "~3.26.1",
     "ember-cli-dependency-checker": "^3.2.0",
-    "ember-cli-inject-live-reload": "2.1.0",
+    "ember-cli-inject-live-reload": "^2.1.0",
     "ember-cli-mirage": "1.1.6",
-    "ember-cli-sri": "2.1.1",
+    "ember-cli-sri": "^2.1.1",
     "ember-cli-terser": "^4.0.2",
     "ember-code-snippet": "2.3.1",
     "ember-data": "3.26.0",
-    "ember-disable-prototype-extensions": "1.1.3",
-    "ember-export-application-global": "2.0.1",
+    "ember-disable-prototype-extensions": "^1.1.3",
+    "ember-export-application-global": "^2.0.1",
     "ember-load-initializers": "^2.1.2",
-    "ember-modifier": "^3.0.0",
+    "ember-maybe-import-regenerator": "^0.1.6",
     "ember-page-title": "^6.2.2",
     "ember-qunit": "^5.1.4",
-    "ember-resize-observer-service": "^0.4.5",
-    "ember-resolver": "8.0.2",
+    "ember-resolver": "^8.0.2",
     "ember-source": "~3.26.2",
-    "ember-source-channel-url": "3.0.0",
+    "ember-source-channel-url": "^3.0.0",
     "ember-template-lint": "^3.6.0",
-    "ember-try": "1.4.0",
+    "ember-try": "^1.4.0",
     "eslint": "^7.32.0",
     "eslint-config-prettier": "^8.3.0",
     "eslint-plugin-ember": "^10.5.4",
     "eslint-plugin-node": "^11.1.0",
     "eslint-plugin-prettier": "^3.4.1",
     "eslint-plugin-qunit": "^6.2.0",
-    "loader.js": "4.7.0",
+    "loader.js": "^4.7.0",
     "npm-run-all": "^4.1.5",
     "prettier": "^2.3.2",
     "qunit": "^2.16.0",
-    "qunit-dom": "^2.0.0"
+    "qunit-dom": "^1.6.0"
```

Comment:
Why'd you downgrade this?

Reply:
I think this was an experiment that was accidentally committed.